### PR TITLE
new keyboard rb1

### PR DESCRIPTION
**Pull request by ryanbaekr**
## Description

Added new keyboard: ryanbaekr rb1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/ryanbaekr/rb1/info.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "maintainer": "ryanbaekr",
+    "usb": {
+        "vid": "0x7262",
+        "pid": "0x0001",
+        "device_version": "0.0.3"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"x":0, "y":0}
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb1/info.json:14]:
```suggestion
                {"matrix": [0, 0], "x":0, "y":0}
```
rb1.[ch] can be deleted with this :)

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ryanbaekr/rb1/config.h [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 1
+
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb1/config.h:25]:
```suggestion
```

File: keyboards/ryanbaekr/rb1/config.h [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * DIRECT: AVR pins used for keys
+ *
+ */
+#define DIRECT_PINS {{ B1 }}
+#define UNUSED_PINS
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb1/config.h:34]:
```suggestion
```

File: keyboards/ryanbaekr/rb1/config.h [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+//#define SOFT_SERIAL_PIN D0  // or D1, D2, D3, E6
+
+
+
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb1/config.h:42]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**